packages/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM centos:7
+
+# Version of calico-mesos plugin
+ENV CALICO_MESOS_VERSION v0.1.1
+ENV CALICO_DOCKER_VERSION v0.8.0
+
+# Install dependencies
+RUN yum install -y epel-release wget rpm-build
+#RUN yum groupinstall -y "Development Tools"
+#RUN yum install -y tar wget curl docker python-devel python-pip git libffi-devel libssl-devel
```

Comment:
If these libraries are necessary, we should add them as rpm linked dependencies so yum gets them for the user